Makefile

```diff
@@ -177,6 +177,7 @@ lint-sol: ## run linter for Solidity
 .PHONY: lint-rust
 lint-rust: ## run linter for Rust
 	cargo fmt --check
+	#cargo clippy -- -Dwarnings			# cannot be used unless all deprecation notices are removed as well
```

Comment:
Can we ignore deprecations warnings just in the CI for now? And have a TODO to re-enable them once resolved

Reply:
Follow up PR removing deprecated code will contain this change.